src/Bowling.js

```diff
@@ -0,0 +1,92 @@
+'use srict'
+
+// var Frame = require('../src/frame')
+function Frame (first, second) {
```

Comment:
Usually a good idea to keep one constructor function/prototype definition to a single file.